.gitignore

```
@@ -1,4 +1,6 @@
 **/target
 **/node_modules
 .idea
-chariot-core/Cargo.lock # https://doc.rust-lang.org/cargo/faq.html#why-do-binaries-have-cargolock-in-version-control-but-not-libraries
\ No newline at end of file
+chariot-core/Cargo.lock # https://doc.rust-lang.org/cargo/faq.html#why-do-binaries-have-cargolock-in-version-control-but-not-libraries
+chariot-client/Cargo.lock
+chariot-server/Cargo.lock
```

```suggestion
Cargo.lock
```

we can just globally omit all `Cargo.lock`s